src/CheckoutSdk.SampleApp/Views/Payment/NonThreeDsFailure.cshtml

```diff
@@ -0,0 +1,84 @@
+﻿@model Checkout.Payments.PaymentProcessed
+
+@{
+    ViewData["Title"] = Model.ResponseSummary;
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h2>Failure</h2>
+
+<div>
+    <h4>PaymentProcessed</h4>
+    <hr />
+    <dl class="dl-horizontal">
+        <dt>
+            @Html.DisplayNameFor(model => model.Id)
```

Comment:
You might you can simply do Html.EditorFor(model => model) and have it render everything

Reply:
Tried it and it would make the app look even more ugly